builds/misc/images.yaml

```diff
@@ -181,6 +181,29 @@ jobs:
           imageName: azureiotedge-c2dmessage-tester
           project: CloudToDeviceMessageTester
 
+################################################################################
+  - job: linux_API_proxy_module
+################################################################################
+    timeoutInMinutes: 80
+    displayName: linuxAPIProxy
+    pool:
+      vmImage: 'ubuntu-16.04'
+    steps:
+      - bash: 'docker login $(registry.address) --username $(registry.user) --password $(registry.password)'
+        displayName: 'Docker Login'
+      # Build API Proxy executable
+      - template: templates/build-api-proxy.yaml
+      # Build API Proxy Image
+      - template: templates/image-linux.yaml
+        parameters:
+          name: API Proxy
+          imageName: api-proxy-module
+          project: api-proxy-module      
+      # Check API proxy
+      - task: ComponentGovernanceComponentDetection@0
+        inputs:
+          dockerImagesToScan: '$(registry.address)/microsoft/api-proxy-module:$(Build.BuildNumber)-linux-amd64,$(registry.address)/microsoft/api-proxy-module:$(Build.BuildNumber)-linux-arm32v7,$(registry.address)/microsoft/api-proxy-module:$(Build.BuildNumber)-linux-arm64v8' 
```

Comment:
I am curious, how this works since EdgeBuilds is private container registry, how does it access to those images?

Reply:
Bear added docker login step for private container registry at the beginning of the job.